src/plugins/intel_cpu/src/transformations/transformation_pipeline.cpp

```diff
@@ -433,7 +433,7 @@ void Transformations::PreLpt(const std::vector<ov::element::Type>& defaultPrecis
     CPU_SET_CALLBACK_COMMON(manager, nmsCallback, ov::pass::ConvertNMS9ToNMSIEInternal);
     CPU_SET_CALLBACK_COMMON(manager, nmsCallback, ov::pass::ConvertMulticlassNmsToMulticlassNmsIE);
     CPU_SET_CALLBACK_COMMON(manager, nmsCallback, ov::pass::ConvertMatrixNmsToMatrixNmsIE);
-    CPU_SET_CALLBACK_COMMON(manager,
```

Comment:
I assume line `CPU_DISABLE_PASS_X64(manager, ov::pass::ScaledDotProductAttentionDecomposition);` also should be deleted. Otherwise you disable decomposition unconditionally for X64

Reply:
Updated, removed.